ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml

```diff
@@ -2778,16 +2778,17 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-vsphere-ipi-ovn-etcd-encryption
-- as: vsphere-8-upi-p2-f14
-  cron: 19 17 14,29 * *
+- as: vsphere-8-ipi-ovn-winc-p2-f14
```

Comment:
Perhaps adding a new test for winc ?

Reply:
yes , I added this , not sure , any new tags are required .

Reply:
I mean add a new test for `vsphere-8-ipi-ovn-winc-p2-f14`. Currently it replace the test `vsphere-8-upi-p2-f14`

Reply:
got it made the update , thanks